Challenge_01/index.html

```diff
@@ -1,104 +1,107 @@
                     </div>
-                    <div class="LastName">
+                    <div class="lastName">
+                        <label for="lname">
                         <div class="title">
                             <p>LastName
-                            <p class="star">*</p>
+                            <span class="star">*</span>
                             </p>
                         </div>
-                        <input type="text">
+                        </label>
+                        <input type="text" id="lname" name="lname">
                     </div>
                 </div>
-                <div class="Email-container">
+                <div class="email-container">
+                    <label for="email">
                     <div class="title">
-                        <p>Email Address
-                        <p class="star">*</p>
-                        </p>
-                    </div>
-                    <input type="text">
+                            <p>Email Address
+                                <span class="star">*</span>
+                            </p>
+                        </div>
+                    </label>
+                    <input type="email" id="email" name="email" autocomplete="email">
                 </div>
 
 
-                <div class="QueryType-continer">
-                    <div class="title">
-                        <p>Query Type
-                        <p class="star">*</p>
-                        </p>
-                    </div>
+                <div class="querytype-continer">
+                        <div class="title">
+                            <p>Query Type
+                                <span class="star">*</span>
+                            </p>
+                        </div>
                     <div class="query-main">
                         <div class="query1">
-
-
-                            <input type="radio" id="">
-
+                            <input type="radio" id="query1" name="query">
+                            <label for="query1">
                             <span for="">General Enquiry</span>
+                        </label>
                         </div>
                         <div class="query2">
-                            <input type="radio">
-
+                            <input type="radio" id="query2" name="query">
+                            <label for="query2">
                             <span for="">Support Request</span>
+                            </label>
                         </div>
                     </div>
 
 
                 </div>
 
-                <div class="Message-Constainer">
+                <div class="message-constainer">
+                    <label for="description-msg">
                     <div class="title">
                         <p>Message
-                        <p class="star">*</p>
+                        <span class="star">*</span>
```

Comment:
inconsistent code formatting
install prettier and set it to format your code whenever you save the file